sdk/python/Makefile

```diff
@@ -29,24 +29,20 @@ ensure::
 	$(PIP) install pipenv==$(PIPENV_VERSION) --user
 	pipenv $(PIPENV_PYTHON_VERSION) install --dev
 	mkdir -p $(PYENVSRC)
-	
+
 build::
 	rm -rf $(PYENVSRC) && cp -R ./lib/. $(PYENVSRC)/
```

Comment:
out of scope, but `rm -rf <unquoted expansion>` is a bit scary

Reply:
Fixed it since I was in the area.